boards/shields/giga_display_shield/boards/arduino_giga_r1_m7.conf

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 Arduino SA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MEMC=y
+CONFIG_STM32_LTDC_RGB565=y
+CONFIG_HEAP_MEM_POOL_SIZE=65536
```

Comment:
heap should not be set in a shield, can add a new Kconfig symbol in a `Kconfig` file e.g. `HEAP_MEM_POOL_ADD_SIZE_GIGA_DISPLAY_SHIELD` and use that

Reply:
Removed.